x/erc20/keeper/ibc_callbacks.go

```diff
@@ -82,41 +68,59 @@ func (k Keeper) OnRecvPacket(
 		data.Denom, data.Amount,
 	)
 
-	// check if the coin is a native staking token
-	bondDenom := k.stakingKeeper.BondDenom(ctx)
-	if coin.Denom == bondDenom {
-		// no-op, received coin is the staking denomination
+	pairID := k.GetTokenPairID(ctx, coin.Denom)
+	pair, _ := k.GetTokenPair(ctx, pairID)
+	// Case 1 - Coin is native EVMOS
+	if !pair.Enabled || pair.IsNativeCoin() {
+		// no-op: continue with the rest of the stack without registration
 		return ack
 	}
```

Comment:
currently, all native coins will be returned with ACK. Case 3 will never be triggered

Reply:
Can replace with something like this instead?
```go
if !pair.Enabled || pair.Denom == utils.BaseDenom { 
           // no-op: continue with the rest of the stack without registration
          return ack
}
```